diff --git a/node_modules/@shopify/react-native-skia/android/build.gradle b/node_modules/@shopify/react-native-skia/android/build.gradle
index 2d71cac..b0cf2e0 100644
--- a/node_modules/@shopify/react-native-skia/android/build.gradle
+++ b/node_modules/@shopify/react-native-skia/android/build.gradle
@@ -186,14 +186,14 @@ dependencies {
     //noinspection GradleDynamicVersion
     extractJNI("com.facebook.fbjni:fbjni:0.2.2")
 
-    if (!sourceBuild) {
-        def rnAarMatcher = "**/react-native/**/*${buildType}.aar"
-        if (REACT_NATIVE_VERSION < 69) {
-            rnAarMatcher = "**/**/*.aar"
-        }
-        def rnAAR = fileTree("${nodeModules}/react-native/android").matching({ it.include rnAarMatcher }).singleFile        
-        extractJNI(files(rnAAR))
-    }
+    //if (!sourceBuild) {
+    //    def rnAarMatcher = "**/react-native/**/*${buildType}.aar"
+    //    if (REACT_NATIVE_VERSION < 69) {
+    //        rnAarMatcher = "**/**/*.aar"
+    //    }
+    //    def rnAAR = fileTree("${nodeModules}/react-native/android").matching({ it.include rnAarMatcher }).singleFile        
+    //    extractJNI(files(rnAAR))
+    //}
 }
 
 afterEvaluate { project ->
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/api b/node_modules/@shopify/react-native-skia/android/cpp/api
index 8fe8155..bf4e8c7 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/api
+++ b/node_modules/@shopify/react-native-skia/android/cpp/api
@@ -1 +1 @@
-/private/var/folders/56/j55p1lwx68q_0b5f_rf1y3940000gn/T/tmp-87309LfNetJOBNYHT/node_modules/@shopify/react-native-skia/cpp/api
\ No newline at end of file
+/Users/andreas/Projects/guess-the-repo/node_modules/@shopify/react-native-skia/cpp/api
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/jsi b/node_modules/@shopify/react-native-skia/android/cpp/jsi
index 2a1b093..978cba6 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/jsi
+++ b/node_modules/@shopify/react-native-skia/android/cpp/jsi
@@ -1 +1 @@
-/private/var/folders/56/j55p1lwx68q_0b5f_rf1y3940000gn/T/tmp-87309LfNetJOBNYHT/node_modules/@shopify/react-native-skia/cpp/jsi
\ No newline at end of file
+/Users/andreas/Projects/guess-the-repo/node_modules/@shopify/react-native-skia/cpp/jsi
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/rnskia b/node_modules/@shopify/react-native-skia/android/cpp/rnskia
index 28d9233..cd14957 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/rnskia
+++ b/node_modules/@shopify/react-native-skia/android/cpp/rnskia
@@ -1 +1 @@
-/private/var/folders/56/j55p1lwx68q_0b5f_rf1y3940000gn/T/tmp-87309LfNetJOBNYHT/node_modules/@shopify/react-native-skia/cpp/rnskia
\ No newline at end of file
+/Users/andreas/Projects/guess-the-repo/node_modules/@shopify/react-native-skia/cpp/rnskia
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/skia b/node_modules/@shopify/react-native-skia/android/cpp/skia
index 3ddfcca..64fb530 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/skia
+++ b/node_modules/@shopify/react-native-skia/android/cpp/skia
@@ -1 +1 @@
-/private/var/folders/56/j55p1lwx68q_0b5f_rf1y3940000gn/T/tmp-87309LfNetJOBNYHT/node_modules/@shopify/react-native-skia/cpp/skia
\ No newline at end of file
+/Users/andreas/Projects/guess-the-repo/node_modules/@shopify/react-native-skia/cpp/skia
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/android/cpp/utils b/node_modules/@shopify/react-native-skia/android/cpp/utils
index 179863e..17ff9e4 120000
--- a/node_modules/@shopify/react-native-skia/android/cpp/utils
+++ b/node_modules/@shopify/react-native-skia/android/cpp/utils
@@ -1 +1 @@
-/private/var/folders/56/j55p1lwx68q_0b5f_rf1y3940000gn/T/tmp-87309LfNetJOBNYHT/node_modules/@shopify/react-native-skia/cpp/utils
\ No newline at end of file
+/Users/andreas/Projects/guess-the-repo/node_modules/@shopify/react-native-skia/cpp/utils
\ No newline at end of file
